lib/person.rb

```diff
@@ -3,14 +3,15 @@
 # Represents a person with a name, age, and parent permission status.
 class Person < Nameable
   attr_reader :id
-  attr_accessor :name, :age, :parent_permission
+  attr_accessor :name, :age, :parent_permission, :rentals
 
   def initialize(name = 'Unknown', age = 0, parent_permission: true)
     super()
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
```

Comment:
* Kindly ensure `A method for adding rentals  be present` which shows the relationship `* A person has many rentals`
* Link to the example in a previous lesson: https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship
* [Check requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_associations.md#:~:text=Create%20the%20has%2Dmany%20side%20of%20Person%20and%20Rental%20(a%20person%20has%20many%20rentals).)

Reply:
Thank you so much @Davidosky007